west.yml

```diff
@@ -50,7 +50,8 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 44ede1e459c061fb1f7b65431535d6b66654ec78
+      revision: pull/910/head
+      # revision: c637477892afef5e2186fe63463703aaab9c53b3
```

Comment:
be very careful here, cause you don't know what the SHA will be when https://github.com/nrfconnect/sdk-zephyr/pull/910 is merged.
but this is a real SHA and could (until garbage collected) be fetched by `west`, and therefor easily make it into `sdk-nrf/main`.

That is a risk I don't like, so I suggest to remove this line.